xargs

```diff
@@ -1,8 +1,16 @@
-# To Find all file name ending with .pdf and remove them
-find -name *.pdf | xargs rm -rf
+---
+syntax: bash
+---
 
-# if file name contains spaces you should use this instead
-find -name *.pdf | xargs -I{} rm -rf '{}'
+# Find all files whose names end with .pdf and remove them
+find -type f -name *.pdf | xargs rm -f
```

Comment:
```suggestion
# Find all files whose names end with .pdf and remove them:
find -type f -name '*.pdf' | xargs rm -f
```
Mandatory fix: [missing trailing colon](https://github.com/cheat/cheatsheets#format).
Optional fix: `-name` argument can be quoted to protect from globbing.